.circleci/config.yml

```diff
@@ -2,14 +2,14 @@ version: 2.1
 jobs:
   lint:
     docker:
-      - image: salsify/ruby_ci:2.5.8
+      - image: salsify/ruby_ci:2.7.2
```

Comment:
I'm not sure it actually matters but we typically run the linter with the oldest supported version of Ruby